frontend/test/metabase/scenarios/visualizations/drillthroughs/chart_drill.cy.spec.js

```diff
@@ -153,6 +155,48 @@ describe("scenarios > visualizations > drillthroughs > chart drill", () => {
     cy.contains("Created At between").click();
   });
 
+  it.skip("should drill-through on filtered aggregated results (metabase#13504)", () => {
+    // go straight to "orders" in custom questions
+    cy.visit("/question/new?database=1&table=2&mode=notebook");
+    cy.findByText("Summarize").click();
+    cy.findByText("Count of rows").click();
+    cy.findByText("Pick a column to group by").click();
+    cy.findByText("Created At").click();
+
+    // add filter: Count > 1
+    cy.findByText("Filter").click();
+    popover().within(() => {
+      cy.findByText("Count").click();
+      cy.findByText("Equal to").click();
+    });
+    cy.findByText("Greater than").click();
+    cy.findByPlaceholderText("Enter a number")
+      .click()
+      .type("1");
+    cy.findByText("Add filter").click();
+
+    // Visualize: line
+    cy.findByText("Visualize").click();
+    cy.findByText("Visualization").click();
+    cy.get(".Icon-line").click();
+    cy.findByText("Done").click();
+    cy.log("**Mid-point assertion**");
+    cy.contains("Count by Created At: Month");
+    // at this point, filter is still present in the page
+    cy.contains("Count is greater than 1");
+
+    // drill-through
+    cy.get(".dot")
+      .eq(10) // random dot
+      .click({ force: true });
+    cy.findByText("View these Orders").click();
+
+    cy.log("**Reproduced on 0.34.3, 0.35.4, 0.36.7 and 0.37.0-rc2**");
+    // when the bug is present, filter is missing name
+    cy.contains("Count is greater than 1");
```

Comment:
It seems correct that it's replacing the Count filter - if not, that's covered in 10767 et al

But the assertion should probably be `cy.contains("Count is equal to 256");` (expecting 256 is the "random dot")

Reply:
Fixed it.
Good catch, and a silly misunderstanding on my part. Thank you.